### ISSUE TITLE
Tabs on mobile announce em-dash on contents list

### ISSUE THREAD

**Issue by NickColley** [alphagov/govuk-frontend#7419]
## Description of the issue

When using Talkback on Android and using the Tabs component (which collapses to use a Contents list on smaller screens) the each list item announced the em-dash as well which could create confusion for users.

Ideally this should be a presentational element, similar to how real bullet points work.

## Steps to reproduce the issue

1. Go to a page with Tabs component on it
2. Activate Talkback 
3. Read the content list

## Actual vs expected behaviour

Actual:
Em-dashes are being announced

Expected:
Only the content of the list should be announced

## Original reports

> Hope you are well. We recently had an accessibility audit and an issue was raised around confusing announcements associated with the tabs component which I wanted to make you aware of. When viewing the tabs on a mobile device, a screen reader reads outs the 'em-dash' which is displayed beside the tab name. 

 _Originally posted by @LucyGeddis in [#100](https://github.com/alphagov/govuk-design-system-backlog/issues/100#issuecomment-5604724373)_

> These are the details for the mobile devices and screen readers used:
> 
> - Android 15 – Samsung Galaxy S22 – Chrome 150 – TalkBack
> - iOS 26 – iPhone 13 – Safari 26 - VoiceOver

 _Originally posted by @LucyGeddis in [#100](https://github.com/alphagov/govuk-design-system-backlog/issues/100#issuecomment-5621759226)_

**Comment by MartinJJones** [comment]
Just in case it helps, I think we fixed a similar issue to this a couple of years back, further info in this PR - https://github.com/alphagov/govuk_publishing_components/pull/3907

Just let me know if you need any further info 🙂

**Comment by romaricpascal** [comment]
We could also look into whether the browser support for [alternative text in the `content` property is strong enough now](https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Properties/content#alternative_text_string_counter_attr).

That being said, I'd like to gently challenge using a character for what effectively is drawing on the screen. Using a border or clip path for doing so and an empty `content` would effectively sidestep the need for alternative text.

**Comment by NickColley** [comment]
In terms of is it appropriate to use a character I think it has a lot of benefits e.g. if the font didnt render the em-dash would work nicely with the fallback, if the font is resized or modified it'll render well, typographers did all the hard work to make these glyphs look great alongside each other so we shouldnt under-estimate that!

That said, it may still be worth drawing it like you say given the access issues if we cant think of another way that's non-breaking.
